go-apps/meep-gis-engine/server/gis-engine.go

```diff
@@ -0,0 +1,1091 @@
+/*
+ * Copyright (c) 2019  InterDigital Communications, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	dataModel "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-data-model"
+	log "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-logger"
+	mod "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-model"
+	mq "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-mq"
+	postgis "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-postgis"
+	sbox "github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-sandbox-ctrl-client"
+	"github.com/gorilla/mux"
+)
+
+const moduleName = "meep-gis-engine"
+const redisAddr = "meep-redis-master.default.svc.cluster.local:6379"
+const sboxCtrlBasepath = "http://meep-sandbox-ctrl/sandbox-ctrl/v1"
+const postgisUser = "postgres"
+const postgisPwd = "pwd"
+
+const (
+	AutoTypeMovement   = "MOVEMENT"
+	AutoTypeMobility   = "MOBILITY"
+	AutoTypeNetChar    = "NETWORK-CHARACTERISTICS-UPDATE"
+	AutoTypePoaInRange = "POAS-IN-RANGE"
+)
+
+type Asset struct {
+	allocated bool
```

Comment:
It's unclear what allocated means for an asset.
Add a comment or use clearer term.